R/get_stan_data.R

```diff
@@ -228,14 +423,21 @@ get_stan_data <- function(input_count_data,
     arg = "infection to count delay"
   )
 
-  validate_both_datasets(
-    input_count_data,
-    input_ww_data,
-    calibration_time = calibration_time,
-    forecast_date = forecast_date
-  )
-
+  # Validate both datasets if both are used----------------------------------
+  if (include_ww == 1) {
+    validate_both_datasets(
+      input_count_data = input_count_data,
+      input_ww_data = input_ww_data,
+      date_time_spine = date_time_spine,
+      lab_site_site_spine = lab_site_site_spine,
+      site_subpop_spine = site_subpop_spine,
+      lab_site_subpop_spine = lab_site_subpop_spine,
+      calibration_time = calibration_time,
+      forecast_date = forecast_date
+    )
+  }
```

Comment:
is there count data validation we can do when `include_ww == 0`?

Reply:
It already happens in the pre-processing of the count data

Reply:
This is just an additional one that is only about how they interact with each other

Reply:
https://github.com/CDCgov/ww-inference-model/issues/197